README.md

```diff
@@ -1 +1,69 @@
-# datatable
\ No newline at end of file
+![Linting](https://github.com/crossroadscx/datatable/actions/workflows/linting.yml/badge.svg) ![Chromatic](https://github.com/crossroadscx/datatable/actions/workflows/chromatic.yml/badge.svg)
+# React Table DataTable
+
+This project layers a UI on top of the every popular [React Table](https://react-table.tanstack.com/) library. The intention is to have a drop-in UI solution for most table use cases, while including simple options to override things like theming, sub-components, and actions.
```

Comment:
May be we can additional info like:

- We are using our own CSS? 
- CSS frameworks can be used with this component?
- Has more dependencies besides `react table?`

Reply:
Great suggestion. Will add

Reply:
Resolved in 990c310d84e844d64909725416c51acd2cd89787

It does have more dependencies than react-table, but they can see those in the `package.json` file. I don't think we need to include those explicitly here. The reason for including `React Table` is that this is a light UI layer on top of the hooks that library provides. It's the main engine for the table.